server/ServerFSM.cpp

```diff
@@ -285,6 +285,18 @@ void ServerFSM::HandleNonLobbyDisconnection(const Disconnection& d) {
                 m_server.DropPlayerEmpireLink(id);
             }
         }
+        // add "player left game" message
+        boost::posix_time::ptime timestamp = boost::posix_time::second_clock::universal_time();
+        std::string data = "[[PLAYER_LEFT_GAME," + player_connection->PlayerName() + "]]";
```

Comment:
arguably the `PLAYER_LEFT_GAME` should be wrapped in `UserStringNoOp` ...

Reply:
Fixed